languages/javascript/reference/keywords/var.md

```diff
@@ -1 +1,30 @@
 # `var`
+
+The `var` keyword declares a [variable](https://developer.mozilla.org/en-US/docs/Glossary/Variable), optionally initializing it to a value.
+```javascript
+var arrow;
+arrow = "->";
+
+// or
+
+var arrow = "->";
+```
+
+The [scope](../info/scope.md) of variables declared using the `var` keyword is either [global](../info/scope.md#global-scope) (if it was not declared in a [function](function.md)) or local to the entire function it was declared in. The main difference between `var` and [`let`](let.md) is that the scope of `var` is the entire function, whereas the scope of `let` is the block it was declared in as well as any contained sub-blocks.
```

Comment:
This is not using the correct link format. Please change to `[title][ref-link]`.